activity-stats/src/components/ActivityStats/ActivityStats.tsx

```diff
@@ -1,9 +1,122 @@
-import data from "../../api-data.json"
+import { useState } from "react";
+import { mockActivityStatsData } from "./mockActivityStatsData";
+import "./ActivityStats.scss";
 
- const ActivityStats = () => {
-    return <>
-    {/* Start here */}
-    </>
-}
+type ActivityData = {
+  type: string;
+  count: number;
+  fiat_currency: string;
+  amount_fiat: string;
+  crypto_currency: string;
+};
 
-export default ActivityStats
\ No newline at end of file
+type CombinedData = {
+  type: string;
+  count: number;
+  fiat_currency: string;
+  amount_fiat: number;
+  crypto_currencies: Set<string>;
+};
+
+// What is count?
+// What constitutes a transaction?
+// fiat currency - can it differ? Does it need to first be converted to dollars and then totalled?
+
+export const ActivityRow = ({ rowData }: { rowData: CombinedData }) => {
+  const { type, count, fiat_currency, amount_fiat, crypto_currencies } =
+    rowData;
+  return (
+    <div className="activity__row" role="listitem">
+      <h3 className="activity__row__label">Total {type}s</h3>
+      <div className="activity__row__total">
+        {formatCurrency(amount_fiat, fiat_currency)}
+      </div>
+      <div className="activity__row__crypto">
+        {crypto_currencies.size === 1
+          ? `${Array.from(crypto_currencies)[0]}`
+          : `${Array.from(crypto_currencies)[0]} + ${
+              crypto_currencies.size - 1
+            } others`}
+      </div>
+      <div className="activity__row__transactions">
+        {count} Transaction{count > 1 ? "s" : ""}
+      </div>
+    </div>
+  );
+};
+
+const ActivityStats = () => {
+  const [activityData] = useState<Map<string, CombinedData>>(() => {
+    return combineDataRows(mockActivityStatsData);
+  });
```

Comment:
Does this need to be state? Can it just be a `const`?

Reply:
Oh yeah, it can probably be a const since it won't need to be updated.